### ISSUE TITLE
Low mAP obtained in testing Simple-baseline model using code in this repo.

### ISSUE THREAD

**Issue by qiuzhongwei-USTB** [leoxiaobin/deep-high-resolution-net.pytorch#15]
I test the simple baseline models downloaded from your link using the code in this repo. I just get 65.3 mAP on pose_resnet50. Flip-test, Ground-truth bbox, are used, and input size is 256*192. GPU: 4 k80, batch-size:32.
And I test onresnet101, 152,  it's still lower than the results in simple-baseline repo.

**Comment by leoxiaobin** [comment]
Could you provide the model's md5 value and the config file? Which pytorch version are you using? Using our pre-trained resnet50 with flip-test and ground-truth bbox, you should achieve a mAP of 72.4.

**Comment by qiuzhongwei-USTB** [comment]
> Could you provide the model's md5 value and the config file? Which pytorch version are you using? Using our pre-trained resnet50 with flip-test and ground-truth bbox, you should achieve a mAP of 72.4.

Here is the config file, pytorch version is 1.0.1, and the pretrained model is downloaded from your link.

AUTO_RESUME: true
CUDNN:
  BENCHMARK: true
  DETERMINISTIC: false
  ENABLED: true
DATA_DIR: ''
GPUS: (0, 1, 2, 3)
OUTPUT_DIR: 'output'
LOG_DIR: 'log'
WORKERS: 24
PRINT_FREQ: 100

DATASET:
  COLOR_RGB: true
  DATASET: 'coco'
  DATA_FORMAT: jpg
  FLIP: true
  NUM_JOINTS_HALF_BODY: 8
  PROB_HALF_BODY: 0.3
  ROOT: 'data/coco/'
  ROT_FACTOR: 45
  SCALE_FACTOR: 0.35
  TEST_SET: 'val2017'
  TRAIN_SET: 'train2017'

MODEL:
  INIT_WEIGHTS: true
  NAME: 'pose_resnet'
  PRETRAINED: 'models/pytorch/imagenet/resnet50-19c8e357.pth'
  TARGET_TYPE: gaussian
  IMAGE_SIZE:
  - 192
  - 256
  HEATMAP_SIZE:
  - 48
  - 64
  SIGMA: 2
  NUM_JOINTS: 17
  EXTRA:
    NUM_LAYERS: 50
    DECONV_WITH_BIAS: false
    NUM_DECONV_LAYERS: 3
    NUM_DECONV_FILTERS: [256, 256, 256]
    NUM_DECONV_KERNELS: [4, 4, 4]
    FINAL_CONV_KERNEL: 1    
LOSS:
  USE_TARGET_WEIGHT: true
TRAIN:
  BATCH_SIZE_PER_GPU: 32
  SHUFFLE: true
  BEGIN_EPOCH: 0
  END_EPOCH: 140
  RESUME: false
  OPTIMIZER: adam
  LR: 0.001
  LR_FACTOR: 0.1
  LR_STEP:
  - 90
  - 120
  WD: 0.0001
  GAMMA1: 0.99
  GAMMA2: 0.0
  MOMENTUM: 0.9
  NESTEROV: true
TEST:
  BATCH_SIZE_PER_GPU: 32
  COCO_BBOX_FILE: 'data/coco/person_detection_results/COCO_val2017_detections_AP_H_56_person.json'
  BBOX_THRE: 1.0
  IMAGE_THRE: 0.0
  IN_VIS_THRE: 0.2
  MODEL_FILE: 'models/pytorch/pose_coco/pose_resnet_50_256x192.pth.tar'
  NMS_THRE: 1.0
  OKS_THRE: 0.9
  USE_GT_BBOX: true
  FLIP_TEST: true
  POST_PROCESS: true
  SHIFT_HEATMAP: true
DEBUG:
  DEBUG: false
  SAVE_BATCH_IMAGES_GT: true
  SAVE_BATCH_IMAGES_PRED: true
  SAVE_HEATMAPS_GT: true
  SAVE_HEATMAPS_PRED: true

**Comment by qiuzhongwei-USTB** [comment]
> Could you provide the model's md5 value and the config file? Which pytorch version are you using? Using our pre-trained resnet50 with flip-test and ground-truth bbox, you should achieve a mAP of 72.4.

0c599c6bf85d7bd211756503ad15f134  pose_resnet_50_256x192.pth

**Comment by leoxiaobin** [comment]
Our pre-trained resnet models are trained using image with BGR channel order, so please set DATASET.COLOR_RGB to False. I found that in your config, you set COLOR_RGB to True. After your correct this, you should get a mAP of 72.4 using our pre-trained resnet50 model.

**Comment by qiuzhongwei-USTB** [comment]
> Our pre-trained resnet models are trained using image with BGR channel order, so please set DATASET.COLOR_RGB to False. I found that in your config, you set COLOR_RGB to True. After you correct this, you should get a mAP of 72.4 using our pre-trained resnet50 model.

Thanks, so HRNet is trained with RGB images, ResNet is trained with BGR images. Is that right?

**Comment by leoxiaobin** [comment]
yes
